lib/client/keystore.go

```diff
@@ -390,6 +393,22 @@ func (fs *FSLocalKeyStore) dirFor(hostname string) (string, error) {
 	return dirPath, nil
 }
 
+// CleanupAll
```

Comment:
comment does not match the function name

`Cleanup cleans up local key store of keys`